crypto/evp/e_sm4.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2017 [Ribose Inc.](https://www.ribose.com). All Rights Reserved.
```

Comment:
As I saw in other files, seems the copyright line should contain company name without the URL (correct me if  I am wrong), the `[]()` likes a markdown syntax...

Reply:
It seems odd to me too.  Better would be:

```
Copyright 2017 Ribose Inc. All Rights Reserved.
```

Reply:
Addressed.